src/TaskModel.vala

```diff
@@ -303,6 +303,30 @@ public class Tasks.TaskModel : Object {
         return false;
     }
 
+    private bool is_default_task_list (E.Source task_list, E.SourceRegistry registry) {
```

Comment:
Just some minor improvements:

If I got this straigt, this function is only useful for GTasks - so how about renaming it to `is_gtasks_default_task_list` and I think we can just throw the Error so its handled by the calling function.

So we end up with something like this:

```
private bool is_gtasks_default_task_list (E.Source task_list, E.SourceRegistry registry) throws Error
```